src/Event/Event.js

```diff
@@ -170,11 +194,19 @@ const Event = ({
       runOnJS(onDragRelease)(translationX, translationY);
     })
     .onFinalize(() => {
-      isDragging.value = false;
+      dragStatus.value = DRAG_STATUS.STATIC;
     });
 
+  /** Wrapper to work with rngh < 2.6.0 */
```

Comment:
did you mean >= 2.6.0, @pdpino ?

Reply:
I meant both >= 2.6.0  and < 2.6.0

I updated the comment, hope now is clearer?

Reply:
it's ok to me, thanks